dynawo/sources/Launcher/dynawo.cmd.in

```diff
@@ -90,6 +90,9 @@ for %%i in ("%tmpVar%") do set "curvesHtml=%%~fi"
 
 if /I "%~1"=="jobs" (
   "%DYNAWO_INSTALL_DIR%"bin\dynawo %2
+) else if /I "%~1"=="generate-preassembled" (
```

Comment:
Did you test this ? I thought elseif did not exist on batch

Reply:
I confirm "else if" works in batch scripts. I may have to fix the if/else conditions to use "else if" by the way

Reply:
I found no trace of else if in the official documentation and I am still not sure it is working with our version or if it will work everywhere

Reply:
I did a quick search on the internet and the scope of the if after an else might be badly defined depending on versions... Better not use it I guess

Reply:
On this website, they use "else if" : https://ss64.com/nt/else.html

Reply:
yes, as I mentioned it exists, but scope of the if is not clearly defined. In the page you mentioned, they do insist on the importance of the location of '(' characters. As Dynawo might be run on multiple versions of Windows and as this condition can be simply written in different if, let's avoid the else if syntax.